pydanticrud/backends/dynamodb.py

```diff
@@ -409,3 +412,31 @@ def save(self, item, condition: Optional[Rule] = None) -> bool:
 
     def delete(self, key):
         self.get_table().delete_item(Key=self._key_param_to_dict(key))
+
+    def batch_save(self, items: list) -> dict:
+        """
+        This function is to write multiple records in to dynamodb and returns unprocessed records in dict
+        if something gone wrong with the record.This will by default try 3 time if any unprocessed
+        records exists in result. Currently, batch_write is not supporting ConditionExpression
+        Refer docs:
+        https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/dynamodb/client/batch_write_item.html
```

Comment:
This description needs to be updated. It no longer tries 3 times and you mention 25 here but the code does 20...it's confusing.